tests/integration_tests/test_server.py

```diff
@@ -20,21 +20,6 @@
 TEST_ADDRESS = ("127.0.0.1", None)
 
 
-async def test_server_deprecated_client(lobby_server):
```

Comment:
Same reason, I'd keep this test. We'll still have some legacy client users which will report the user agent `faf-client`.

Reply:
OK